pkg/controller/backup_finalizer_controller.go

```diff
@@ -100,14 +102,6 @@ func (r *backupFinalizerReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
-	switch backup.Status.Phase {
-	case velerov1api.BackupPhaseFinalizing, velerov1api.BackupPhaseFinalizingPartiallyFailed:
-		// only process backups finalizing after  plugin operations are complete
-	default:
-		log.Debug("Backup is not awaiting finalizing, skipping")
-		return ctrl.Result{}, nil
-	}
-
```

Comment:
Are we sure we want to remove this? Keeping it might be a good guard against a backup somehow getting into the queue or a backup that got updated out of this phase some time between being queued and processed. I think it won't hurt to keep this check here to make sure we never process a backup here that's not in a Finalizing phase.

Reply:
Ok will revert this.